docs/writing/posts/consulting.md

```diff
@@ -0,0 +1,207 @@
+---
+draft: False
+date: 2024-01-22
+slug: consulting
+categories:
+  - Personal
+authors:
+  - jxnl
+---
+
+# What I Learned from Indie Consulting
+
+!!! note "Fragments"
+
+    This is just a note to the reader that I like to write in a fragmented style using speech to text so if you see any errors or don't really understand what's going on feel free to just contribute and edit or leave a question on the github repo.
+
+I specify indie consulting as something that is completely and wholly separate from the big-time consulting we hear about from those ridiculous institutions. Check out this video roasting [McKinsey](https://www.youtube.com/watch?v=AiOUojVd6xQ) From John Oliver to understand how I feel about many of these folks. Theres another great [video](https://www.youtube.com/watch?v=vZE0j_WCRvI) that i saw on tiktok.
+
+<!-- more -->
+
+!!! warning "The failure of big time consulting"
+
+    The difference between Indie Consulting and Big Time Consulting as I define it can be captured in one simple idea. Something that Steve Jobs once talked about:
```

Comment:
```suggestion
    The difference between Indie Consulting and Big Time Consulting as I define it can be captured in one simple idea. Something that [Steve Jobs once talked about](https://www.youtube.com/watch?v=-c4CNB80SRc):
```